hooks/conan-center.py

```diff
@@ -76,22 +82,26 @@ def _get_message(self, message):
             return message
 
     def success(self, message):
-        self._output.success(self._get_message(message))
+        if self._logging_level < WARNING:
```

Comment:
Do we care about success messages?
```suggestion
        if self._logging_level <= INFO:
```